string_validator.rb

```diff
@@ -0,0 +1,37 @@
+require 'pry'
```

Comment:
Watch your indentation size. Can you think of any other things that can help you other than your longish if then statement?